example/offboard_velocity/offboard_velocity.cpp

```diff
@@ -67,10 +67,8 @@ inline void offboard_log(const std::string &offb_mode, const std::string msg)
  *
  * returns true if everything went well in Offboard control, exits with a log otherwise.
  */
-bool offb_ctrl_ned(Device &device)
+bool offb_ctrl_ned(Offboard* offboard)
```

Comment:
You're passing the smart pointer `std::shared_ptr<Offboard> offboard` here.

Reply:
Yoiks.